src/utils/crypto.js

```diff
@@ -65,7 +65,14 @@ const Crypto = {
         let signature = Axlsign.sign(privateKeyBytes, dataBytes, SecureRandom.default.randomUint8Array(64));
         return Base58.encode(signature);
     },
-    isValidTransactionSignature: function (dataBytes, signature, publicKey) {
+    isValidTransactionSignature: function (transaction, signature, publicKey) {
+        if (!transaction || typeof transaction !== 'object') {
```

Comment:
'object' is meaningless. We need to check whether this object has `toBytes()` method or not. change this statement to `!(transaction instanceof Transaction)`